node/network/statement-distribution/src/lib.rs

```diff
@@ -1061,10 +1064,10 @@ async fn retrieve_statement_from_message<'a>(
 					let is_large_statement = message.is_large_statement();
 
 					match info.available_peers.entry(peer) {
-						Entry::Occupied(mut occupied) => {
+						IEntry::Occupied(mut occupied) => {
 							occupied.get_mut().push(message);
 						}
-						Entry::Vacant(vacant) => {
+						IEntry::Vacant(vacant) => {
```

Comment:
Line: 1078 

`if let Some(sender) = std::mem::take(&mut info.peer_sender)`

should be

`
if let Some(sender) = info.peer_sender.take() {
`